pages/common/logstash.md

```diff
@@ -0,0 +1,12 @@
+# logstash
+
+> Migrate data from one source into another.
```

Comment:
"Migrate" might be a slightly misleading term for it. How about "Tool to consume data from various sources, transform it, and send it to various outputs." ?